models/article-model.js

```diff
@@ -56,7 +56,7 @@ exports.articleExists = ((id, next) => {
 exports.getArticlesQuery = ((next, topic='') => {
     if (topic) {
         // articles filtered by topic
-        return db.query(`SELECT a.article_id, a.title, a.topic, a.author, a.created_at, a.votes, a.article_img_url, COUNT(c.article_id) AS comment_count
+        return db.query(`SELECT a.article_id, a.title, a.topic, a.author, a.created_at, a.votes, a.article_img_url, CAST(COUNT(a.article_id) AS INTEGER) AS comment_count
```

Comment:
Great refactor into type converting with sql rather than using a js map after the fact